### PR TITLE
Bot permission granting in Discord integration doc

### PR DESCRIPTION
**Pull request by anthonymag**
## Proposed change
Explaining in the Discord integration documentation how to adjust the bot's permissions during the initial configuration.
Currently, the documentation's integration link will set the permissions integer to `0` which will not let the bot do anything.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
Hello, and thanks for considering this PR! During my setting up of the Discord integration, I was having trouble getting messages to appear in my Discord and only later discovered it was because the bot lacked permissions. This step is not currently documented which hopefully I can solve.

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/discord.markdown [hunk 1]
```diff
@@ -35,15 +35,21 @@ The name you give your application on the [Discord My Apps page](https://discord
...
   <img src='/images/screenshots/discord-bot.png' />
 </p>
 
-Now use the Discord Authorization page with the **Application ID** of your [application](https://discordapp.com/developers/docs/topics/oauth2#bots).
+Next, decide what permissions your bot will have within your server. Under the 'Bot' section, select the permissions you want to grant and copy the permissions integer from the bottom field.
 
-`https://discordapp.com/api/oauth2/authorize?client_id=[APPLICATION_ID]&scope=bot&permissions=0`
+<p class='img'>
+  <img src='/images/screenshots/discord-bot-permissions.png' />
+</p>
```
Comment (dannytsang) [inline source/_integrations/discord.markdown]: Please change to Markdown image syntax instead of HTML.

**Comment by dannytsang** [comment]
I don't seem to have an issue with this. My bot app is set to 0 and I am still getting posts in Discord. Anyone else confirm this?

**Review by dannytsang — CHANGES_REQUESTED** [review]

File: source/_integrations/discord.markdown (hunk 1 above)
Comment (anthonymag) [reply source/_integrations/discord.markdown]: Thanks, done and went ahead and updated the other images as well.

**Comment by anthonymag** [comment]
> I don't seem to have an issue with this. My bot app is set to 0 and I am still getting posts in Discord. Anyone else confirm this?

That is odd; I can say that changing this one step finally got it working for me whereas before I was having nothing post to my channels.

**Comment by dannytsang** [comment]
Changes look good.

@frenck could you advise what to do in this case where there's a difference in setup (see above) please.

**Comment by frenck** [comment]
> Changes look good.
> 
> @frenck could you advise what to do in this case where there's a difference in setup (see above) please.

It really depends on what permissions you want your bot the have on the server. I think this makes sense in a way, as it helps with giving it whatever permission they prefer.

I don't see a direct blocker to not merge this in 👍 

../Frenck

**Review by dannytsang — APPROVED** [review]
Thanks @anthonymag

**Pull request merged**